python-threatexchange/threatexchange/extensions/vpdq/vpdq_util.py

```diff
@@ -13,6 +13,8 @@
 VPDQ_TIMESTAMP_PRECISION = 3
 VPDQ_QUALITY_THRESHOLD = 50
 VPDQ_DISTANCE_THRESHOLD = 31
+VPDQ_QUERY_MATCH_THRESHOLD_PERCENT = 0.0
```

Comment:
blocking: In the production the value of this 80%. 0.0 and 0.0 means to match everything even if zero frames overlap.